com.unity.render-pipelines.high-definition/Editor/Lighting/LightUnit/LightUnitSlider.cs

```diff
@@ -0,0 +1,316 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.HighDefinition;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    class LightUnitSlider
+    {
+        static class SliderConfig
+        {
+            public const float k_IconSeparator      = 6;
+            public const float k_MarkerWidth        = 4;
+            public const float k_MarkerHeight       = 2;
+            public const float k_MarkerTooltipScale = 4;
+            public const float k_ThumbTooltipSize   = 10;
+        }
+
+        protected LightUnitSliderUIDescriptor m_Descriptor;
```

Comment:
The descriptor is not meant to be modified, so you should add a `readonly`  keyword for safety.